src/libraries/System.Net.Http/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -3,6 +3,9 @@
     <type fullname="System.Net.Http.DiagnosticsHandler">
       <method signature="System.Boolean IsGloballyEnabled()" body="stub" value="false" feature="System.Net.Http.EnableActivityPropagation" featurevalue="false" />
     </type>
+    <type fullname="System.Net.Http.Metrics.MetricsHandler">
```

Comment:
Can we use `[FeatureSwitchDefinition]` instead?

https://github.com/dotnet/runtime/blob/6c4f46b27f44d24bd36d0beb50635d266de5b520/src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs#L292-L296

Reply:
Should I change the other one for `DiagnosticsHandler` too ?

Reply:
I think it's important that the `<MetricsSupport>` and `<HttpActivityPropagationSupport>` are empty during build of the runtime pack, because we don't want the linker to apply it on runtime pakc. I tested it and it seems fine. I also tested that `MetricsHandler` and `DiagnosticsHandler` are missing from the trimmed app.

Reply:
so, done